mne/io/utils.py

```diff
@@ -342,3 +342,19 @@ def _construct_bids_filename(base, ext, part_idx, validate=True):
     if dirname:
         use_fname = op.join(dirname, use_fname)
     return use_fname
+
+
+def _make_split_fnames(fname, n_splits, split_naming):
+    """Make a list of split filenames."""
+    if n_splits == 1:
+        return [fname]
+    res = []
+    base, ext = op.splitext(fname)
+    for i in range(n_splits):
+        if split_naming == "neuromag":
+            res.append(f"{base}-{i:d}{ext}" if i else fname)
+        elif split_naming == "bids":
+            res.append(_construct_bids_filename(base, ext, i + 1))
+        else:
+            raise NotImplementedError(f"{split_naming=} is not supported")
```

Comment:
For this it's better to use `_check_option` at the top of the function to avoid coverage drops, then instead of the `elif` branch above you can just go straight to `else`:
```
else:
    assert split_naming == "bids"
```
knowing it must succeed (hence the use of bare `assert`, it's a kind of "active comment")

Reply:
It's more of a reminder to add a clause here if a new option for `split_naming` is ever added. But yes `assert` is simpler and doesn't need a test. Changed it.
I also added _check_option to `epochs.save`, like it's done in `mne.io.base:1702`